lib/web_ui/test/canvaskit/overlay_canvas_factory_test.dart

```diff
@@ -13,11 +13,13 @@ void main() {
 }
 
 void testMain() {
-  group('$RenderCanvasFactory', () {
+  group('$OverlayCanvasFactory', () {
```

Comment:
How do we test the two strategies? We want to make sure tests cover single and multi context modes.

Reply:
The only difference between those modes is the factory function which creates the DisplayCanvas. This test is just for the functions of `DisplayCanvas` itself, so it should be agnostic to what kinds of `DisplayCanvas`es are actually made. I refactored the test a bit to clarify that.